lib/loadBackends.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const fs = require('fs');
+
+module.exports = function loadBackends(backendsFile, vars){
+	var backendData = fs.readFileSync(backendsFile, { encoding: 'utf-8' } );
+
+	for(var i=0; i<vars.length; i++) {
+		var key = vars[i];
+		var val = process.env[key];
+		if( val ) {
+			var regex = new RegExp('\\\$\\\{'+ key.trim()+'\\\}', 'gm');
+			backendData = backendData.replace(regex, val);
+		} else {
+			throw new Error(`Environment variable ${key} is required to deploy backends`);
+		}
+	}
+
+	return JSON.parse(backendData);
```

Comment:
ahhh, this means that you can't write the backend data in JavaScript…  https://github.com/Financial-Times/next-cdn/blob/master/src/config/prod.js

maybe as simple as adding a line on line 5:-
```js
if (/\.js$/test(backendsFile) {
  return require(path.join(process.cwd(), backendsFile));
}
```